sbt-scalajs-bundler/src/main/scala/scalajsbundler/sbtplugin/ScalaJSBundlerPlugin.scala

```diff
@@ -557,7 +566,9 @@ object ScalaJSBundlerPlugin extends AutoPlugin {
         addPackages(baseDir, installDir, useYarn.value, log, npmExtraArgs.value, yarnExtraArgs.value)(s"jsdom@$jsdomVersion")
       }
       installDir
-    }
+    },
+
+    requireJsDomEnv := false
```

Comment:
What do you think of defaulting to `requiresDOM.value` instead, to not break existing builds?

Reply:
Make sense! Will do